Polyhedron/demo/Polyhedron/MainWindow.h

```diff
@@ -388,7 +388,7 @@ protected Q_SLOTS:
   void setMenus(QString, QString, QAction *a);
   /// plugin black-list
   QSet<QString> plugin_blacklist;
-  QList<QDir> plugins_directories;
+  QMap<QString, std::vector<QString> > PathNames_map; //For each non-empty plugin directory, contains a vector of plugin names
```

Comment:
There a problem with the naming of that variable. It should follow the CGAL naming convention `path_names_map` (or Qt convention `pathNamesMap`).